frontend/ui/elements/screensaver_menu.lua

```diff
@@ -164,15 +174,20 @@ return {
                     genMenuItem(_("Bottom"), "screensaver_message_position", "bottom", nil, true),
                 },
             },
-            {
-                text = _("Hide reboot/poweroff message"),
```

Comment:
If this is the last item in the array, you can use:
```
Device:canReboot() and Device:canPowerOff() and {
  text...
} or nil
```
instead of the table.insert() below.